src/Symfony/Component/Serializer/Normalizer/AbstractObjectNormalizer.php

```diff
@@ -287,7 +287,7 @@ public function denormalize($data, $class, $format = null, array $context = [])
             try {
                 $this->setAttributeValue($object, $attribute, $value, $format, $context);
             } catch (InvalidArgumentException $e) {
-                throw new NotNormalizableValueException($e->getMessage(), $e->getCode(), $e);
+                throw new NotNormalizableValueException(sprintf('%s. The type of the "%s" attribute for class "%s" seems wrong ("%s" given).', $e->getMessage(), $attribute, $class, \gettype($value)), $e->getCode(), $e);
```

Comment:
I'm not sure about the wording: the type is either wrong or bad, how can it "seem" wrong?
We should see how we construct similar sentences in the code base.